app/lib/actions/katello/repository/clone_yum_content.rb

```diff
@@ -19,6 +21,16 @@ def plan(source_repo, target_repo, filters, purge_empty_units, options = {})
             remove_clauses = clause_gen.remove_clause
           end
 
+          # if we are providing a list of files, process that here and override filters
+          if rpm_filenames.present?
+            # ensure we have all the files we want to copy
+            rpms_available = source_repo.rpms.pluck(:filename)
+            rpm_filenames.each do |filename|
+              fail "%s not available in repository %s" % [filename, source_repo.label] unless rpms_available.include? filename
+            end
+            copy_clauses = { 'filename' => { '$in' => rpm_filenames } }
```

Comment:
The intent here is to override the copy_clauses from above if any are defined? That might be unexpected behavior to a user if they don't get a warning like "you cannot provide filters and a list of RPMs"

Reply:
At least one user wanted a true "override" that ignored filters, so I added a log message when someone has filters + pkglist overrides set.